app/controllers/assignments_controller.rb

```diff
@@ -309,6 +309,38 @@ def batch_runs
     end
   end
 
+  # Return the chart data of assignment grade distributions
+  def grade_distribution_graph_data
+    assignment = Assignment.find(params[:id])
+    labels = []
+    (0..19).each { |i| labels.push((5 * i).to_s + '-' + (5 * i + 5).to_s) }
```

Comment:
Use `.map` instead of `.each`, since the former returns a list, so you don't have to `push` manually.